### PR TITLE
OBSDOCS-1416: Add the Network Observability Operator to the OTEL Troubleshooting page

### PR DESCRIPTION
**Pull request by max-cx**
Version(s):

4.12, 4.13, 4.14, 4.15, 4.16, 4.17, 4.18

Issue:

https://issues.redhat.com/browse/OBSDOCS-1416

Link to docs preview:

https://85689--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-troubleshooting.html#using-network-observability-operator-for-troubleshooting_otel-troubleshoot

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by IshwarKanse — APPROVED** [review]

File: modules/otel-troubleshoot-network-traffic.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+You can debug the traffic between your observability components by visualizing it with the Network Observability Operator.
+
+.Prerequisites
+
+* You have installed the Network Observability Operator as explained in "Installing the Network Observability Operator".
+
+.Procedure
+
+. In the web console, go to *Observe* -> *Network Traffic* -> *Topology*.
```
[the remark this answers is not in the document]
Comment (max-cx) [reply modules/otel-troubleshoot-network-traffic.adoc:17]: TY, sure, I can add the OCP attribute here.

**Review by iblancasa — APPROVED** [review]

**Pull request merged**